examples/AspNet/Global.asax.cs

```diff
@@ -35,7 +35,7 @@ protected void Application_Start()
             this.openTelemetry = Sdk.CreateTracerProviderBuilder()
                  .AddHttpClientInstrumentation()
                  .AddAspNetInstrumentation()
-                 .UseJaegerExporter(c =>
+                 .AddJaegerExporter(c =>
```

Comment:
One minor suggestion, it seems we've been very inconsistent in the variable name, for example we use `c` here, and in other places we use `o`, `opt`, `options` - I would vote for changing them all to `options`.

BTW, `this.openTelmetry` is another thing we probably want to change - it should be called `tracerProvider`.

Not blocking this PR though.